components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/WorkflowImplService.java

```diff
@@ -54,6 +54,16 @@ void addBPSProfile(BPSProfile bpsProfileDTO, int tenantId)
      */
     void removeBPSProfile(String profileName) throws WorkflowImplException;
 
+    /**
+     * Remove all BPS profiles of given tenant id.
+     *
+     * @param tenantId
```

Comment:
Shall we complete the method comments by adding the descriptions?

Reply:
Shall we fix this in other applicable places as well?

Reply:
👍